dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/AbstractCrudController.java

```diff
@@ -662,6 +663,37 @@ public void setAsFavorite( @PathVariable( "uid" ) String pvUid, HttpServletReque
         webMessageService.send( WebMessageUtils.ok( message ), response, request );
     }
 
+    @RequestMapping( value = "/{uid}/subscriber", method = RequestMethod.POST )
```

Comment:
Should that be /subscribe ?

Reply:
I think using nouns are better in REST APIs.